test/python/aer/test_simulator_interfaces.py

```diff
@@ -53,18 +52,14 @@ def test_qasm(self, qe_token, qe_url):
         circuit.cx(qr[1], qr[2])
         circuit.measure(qr, cr)
 
-        sim_cpp = 'qasm_simulator'
-        sim_py = 'qasm_simulator_py'
-        sim_ibmq = 'ibmq_qasm_simulator'
```

Comment:
Why remove the run on the ibm_qasm_simulator in this test? This test is meant to do some cross testing between different simulators to make sure they have the same output for a given input.

Reply:
Because it’s in the aer folder and redone in ibmq

Reply:
I see I left ibmq in the import as I went between keeping and not a few times. :-)

Reply:
oh yeah. good point.